osu.Game/Overlays/Options/General/LoginOptions.cs

```diff
@@ -10,38 +11,72 @@
 
 namespace osu.Game.Overlays.Options.General
 {
-    public class LoginOptions : OptionsSubsection
+    public class LoginOptions : OptionsSubsection, IOnlineComponent
```

Comment:
i want to use interfaces because i believe there are going to be more things that online components need to handle apart from state changes.